query-node/mappings/src/membership.ts

```diff
@@ -392,8 +422,13 @@ export async function members_InvitesTransferred({ store, event }: EventContext
   await store.save<InvitesTransferredEvent>(invitesTransferredEvent)
 }
 
-export async function members_MemberInvited({ store, event }: EventContext & StoreContext): Promise<void> {
-  const [memberId, inviteMembershipParameters, invitedMemberBalance] = new Members.MemberInvitedEvent(event).params
+export async function members_MemberInvited({ store, event, block }: EventContext & StoreContext): Promise<void> {
+  console.log('block', block)
+  const { specVersion } = block.runtimeVersion
+  const { specVersion: v2001, params: asV2001 } = new MemberInvitedEvent_V2001(event)
+  const { params: asV1001 } = new MemberInvitedEvent_V1001(event)
+  const [memberId, inviteMembershipParameters, maybeInvitedMemberBalance] = specVersion === v2001 ? asV2001 : asV1001
```

Comment:
This implementation causes an error when trying to process `MemberInvited` event from the mainnet chain:
```
ERROR: Stopping the proccessor due to errors: {} name: TypeError, message: Cannot read property 'value' of undefined, stack: TypeError: Cannot read property 'value' of undefined
    at Members_MemberInvitedEvent_V2001.get params [as params] (/joystream/query-node/mappings/lib/types/generated/2001/members.js:228:101)
    at members_MemberInvited (/joystream/query-node/mappings/lib/mappings/src/membership.js:201:41)
    at MappingsLookupService.call (/joystream/node_modules/@joystream/hydra-processor/lib/executor/MappingsLookupService.js:80:15)
    at /joystream/node_modules/@joystream/hydra-processor/lib/executor/TransactionalExecutor.js:41:43
    at EntityManager.<anonymous> (/joystream/node_modules/typeorm/entity-manager/EntityManager.js:80:46)
    at step (/joystream/node_modules/tslib/tslib.js:144:27)
    at Object.next (/joystream/node_modules/tslib/tslib.js:125:57)
    at fulfilled (/joystream/node_modules/tslib/tslib.js:115:62)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```
```suggestion
  const [memberId, inviteMembershipParameters, maybeInvitedMemberBalance] =
    specVersion === 2001 ? new MemberInvitedEvent_V2001(event).params : new MemberInvitedEvent_V1001(event).params
```